### ISSUE TITLE
azure_postgresql_flexible_server does not return public network access or similar flag

### ISSUE THREAD

**Issue by ManojDhanorkar** [turbot/steampipe-plugin-azure#821]
Below azure_postgresql_flexible_server does not return public network access  or similar flag. We are looking for some flag which should  indicate if public access is enabled on the server. 

select
  name,
  server_properties
from
  azure_postgresql_flexible_server;

Sample response:  ( server name replaced)

{"AdministratorLogin":"abc","AdministratorLoginPassword":null,"AvailabilityZone":"1","Backup":{"backupRetentionDays":7,"geoRedundantBackup":"Disabled"},"CreateMode":null,"FullyQualifiedDomainName":"[abc.postgres.database.azure.com](http://abc.postgres.database.azure.com/)","HighAvailability":{"mode":"Disabled"},"MaintenanceWindow":{"customWindow":"Disabled","dayOfWeek":0,"startHour":0,"startMinute":0},"MinorVersion":"3","Network":{},"PointInTimeUTC":null,"SourceServerResourceID":null,"State":"Ready","Storage":{"storageSizeGB":128},"Tags":null,"Version":"16"}

Steampipe v0.22.1

Azure 0.64.0

**Comment by ManojDhanorkar** [comment]
Is there an upcoming release which would have this fix ?

**Comment by misraved** [comment]
Yes @ManojDhanorkar, we will be releasing the fix by the end of this week 👍.

**Comment by ManojDhanorkar** [comment]
Thanks

**Comment by ManojDhanorkar** [comment]
will this be available in https://github.com/turbot/steampipe-plugin-azure/releases/tag/v0.65.0 ?

**Comment by misraved** [comment]
Hi @ManojDhanorkar, the fix is now available in `v0.65.0` plugin version 👍 !!
